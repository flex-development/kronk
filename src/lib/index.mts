/**
 * @file Entry Point - Library
 * @module kronk/lib
 */

export { default as Argument } from '#lib/argument'
export { default as Command } from '#lib/command'
export { default as Option } from '#lib/option'
export { default as VersionOption } from '#lib/version.option'
