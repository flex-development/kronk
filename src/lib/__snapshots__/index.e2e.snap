// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`e2e:lib > should expose public api 1`] = `
[
  "Argument",
  "Command",
  "Option",
]
`;
